index.html

```diff
@@ -5,13 +5,206 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Task 1</title>
     <link rel="stylesheet" href="styles.css">
+    <title>Task 1</title>
 </head>
```

Comment:
To add title icon: 
<link rel="icon" type="image/x-icon" href="/images/favicon.ico">